strictdoc/server/routers/main_router.py

```diff
@@ -2696,6 +2698,49 @@ def get_search(q: Optional[str] = None):
             status_code=200,
         )
 
+    # @relation(SDOC-SRS-120, scope=function)
```

Comment:
This @relation tag must be moved inside a """ comment as follows:

```py
def get_autocomplete_results(q: Optional[str] = None):
    """
    @relation(SDOC-SRS-120, scope=function)
    """
````

This is how the source code parser recognizes the function markers in Python.